tests/ltp/patches/ltp_listxattr_listxattr01_fix.patch

```diff
@@ -0,0 +1,49 @@
++ Patch Description: Tests were failing as no xattr support in loop device
++ So modofied the tests to use root filesystem with xattr support
```

Comment:
Are sure that the root file system has xattr support?

Reply:
root file system doesn't have it. That's why we are mounting /dev/vda with this option to be able to run the test.